doc/cassandra_compatibility.md

```diff
@@ -40,6 +40,16 @@ Users should consider to upgrade to one of the latest Cassandra releases togethe
 | 2.0.0 -> latest | Cassandra 3.0.11 | Cassandra 3.0.11 |
 
 
+## ecaudit_c2.2
+
+This flavor is built with the latest version of the Apache Cassandra 2.2.x series.
+The table below list the Cassandra version used while building each ecAudit release and indicate compatibility with other Cassandra versions.
+
+| ecAudit Version | Compiled With    | Compatible With                  |
+| --------------- | ---------------- | -------------------------------- |
+| 2.2.0 -> latest | Cassandra 2.2.14 | Cassandra 2.2.14                 |
```

Comment:
2.2.0 is not released yet... But it will be the first ecAudit-version supporting c2.2